core/src/main/resources/application.properties

```diff
@@ -57,7 +57,6 @@ yona.security.maxUpdateUserAttemptsPerTimeWindow = 2
 yona.security.corsAllowed = true
 yona.security.sslRootCertFile = resources/rootcert.cer
 yona.security.ovpnProfileFile = resources/profile.ovpn
-yona.security.appProvidedPasswordEnabled = true
```

Comment:
Yes, the production server has this as false. That's why I know it can be removed.